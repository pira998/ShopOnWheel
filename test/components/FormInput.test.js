import React from 'react';
import renderer from 'react-test-renderer';
import FormInput from '../../components/FormInput'


test('renders correctly', () => {
  const tree = renderer.create(<FormInput />).toJSON();
  expect(tree).toMatchSnapshot();
});