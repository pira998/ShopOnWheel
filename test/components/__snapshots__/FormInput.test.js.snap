// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={Object {}}
>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#898B9A",
          "fontFamily": "sans-serif",
          "fontSize": 14,
          "lineHeight": 22,
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#FF1717",
          "fontFamily": "sans-serif",
          "fontSize": 14,
          "lineHeight": 22,
        }
      }
    >
      
    </Text>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#F5F5F8",
        "borderRadius": 12,
        "flexDirection": "row",
        "height": 55,
        "marginTop": 8,
        "paddingHorizontal": 24,
      }
    }
  >
    <TextInput
      allowFontScaling={true}
      autoCapitalize="none"
      autoCompleteType="off"
      keyboardType="default"
      placeholderTextColor="#898B9A"
      rejectResponderTermination={true}
      style={
        Object {
          "flex": 1,
        }
      }
      underlineColorAndroid="transparent"
    />
  </View>
</View>
`;
